src/main/java/se/sundsvall/contactsettings/api/model/ContactChannel.java

```diff
@@ -0,0 +1,100 @@
+package se.sundsvall.contactsettings.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import se.sundsvall.contactsettings.api.model.enums.ContactMethod;
+
+@Schema(description = "Contact channel model")
+public class ContactChannel {
+
+	@Schema(description = "Method of contact", example = "SMS")
+	private ContactMethod contactMethod;
+
+	@Schema(description = "Alias for the destination", example = "Privat mobil")
+	private String alias;
+
+	@Schema(description = "Point of destination", example = "0701234567")
+	private String destination;
+
+	@Schema(description = "Signal if channel should be used or not when sending message", example = "true")
+	private boolean send;
```

Comment:
rename to disabled? 

(booleans are false by default and "disabled = false", seems to be a good default)